java/cdk/src/main/java/sleeper/cdk/stack/TransactionLogStateStoreStack.java

```diff
@@ -150,4 +150,12 @@ public void grantDeleteSnapshots(IGrantable grantee) {
         allSnapshotsTable.grantReadWriteData(grantee);
         dataStack.grantReadDelete(grantee);
     }
+
+    public void grantDeleteTransactions(IGrantable grantee) {
+        filesLogTable.grantReadWriteData(grantee);
+        partitionsLogTable.grantReadWriteData(grantee);
+        latestSnapshotsTable.grantReadData(grantee);
+        allSnapshotsTable.grantReadData(grantee);
+        dataStack.grantReadWrite(grantee);
```

Comment:
Should this be dataStack.grantReadDelete?

Reply:
Thanks, fixed